app/src/main/AndroidManifest.xml

```diff
@@ -9,7 +9,9 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
 
-        <activity android:name=".MainActivity">
+        <activity
+            android:name=".MainActivity"
+            android:configChanges="orientation|screenSize">
```

Comment:
If you rotate device fragment changes to default(news) and bottom navigation doesn't change. This line fixes it(left last opened fragment)